app/controllers/admin/greet_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+class Admin::GreetController < AdminApplicationController
```

Comment:
Use similar naming as the user portal.

Reply:
Yeah, I didn't see it very important as I don't think any of the two controllers are to be long lived. Will update.